.github/workflows/run-tests.yml

```
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4.1.0
```

```suggestion
        uses: actions/setup-python@v4
```

`v4` == `v4.*` which seems more like what we actually want.